eventcollectors/src/main/kotlin/events/boudicca/eventcollector/collectors/FamilienkarteEventCollector.kt

```diff
@@ -0,0 +1,67 @@
+package events.boudicca.eventcollector.collectors
+
+import events.boudicca.SemanticKeys
+import events.boudicca.api.eventcollector.Event
+import events.boudicca.api.eventcollector.Fetcher
+import events.boudicca.api.eventcollector.TwoStepEventCollector
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Element
+import java.time.LocalDate
+import java.time.LocalTime
+import java.time.OffsetDateTime
+import java.time.ZoneId
+import java.time.format.DateTimeFormatter
+
+class FamilienkarteCollector : TwoStepEventCollector<String>("familienkarte") {
+
+    private val fetcher = Fetcher()
+
+
+    override fun getAllUnparsedEvents(): List<String> {
+        val document = Jsoup.parse(fetcher.fetchUrl("https://www.familienkarte.at/de/freizeit/veranstaltungen/veranstaltungskalender.html?events_cat_key=8"))
```

Comment:
short question about the url. why limit it to events from category 8? also, i saw that by default the page only shows events from today, we should extend that so we get more events from them